utils/cosmos.ts

```diff
@@ -18,19 +17,60 @@ export const royaltyRateBasisPoints = 1000 // 10% as in current chain config
 export const royaltyFeeAmount = 25000 // 2.5%
 export const royaltyUserAmount = 1000000 - royaltyFeeAmount // 1000000 - fee
 
-let iscnSigningClient: ISCNSigningClient | null = null
+let iscnSigningClient: any = null
+let iscnLib: any = null
+let iscnParsingLib: any = null
+let cosmLib: any = null
+let iscnMsgLib: any = null
+let likenftMsgLib: any = null
+
+async function getISCNLib () {
+  if (!iscnLib) {
+    iscnLib = await import(/* webpackChunkName: "iscn" */ '@likecoin/iscn-js')
+  }
+  return iscnLib
+}
+
+async function getISCNParsingLib () {
+  if (!iscnParsingLib) {
+    iscnParsingLib = await import(/* webpackChunkName: "iscn-parsing" */ '@likecoin/iscn-js/dist/iscn/parsing')
+  }
+  return iscnParsingLib
+}
 
-export async function getSigningClient (): Promise<ISCNSigningClient> {
+async function getISCNMsgLib () {
+  if (!iscnMsgLib) {
+    iscnMsgLib = await import(/* webpackChunkName: "iscn-msg" */ '@likecoin/iscn-js/dist/messages/parsing')
+  }
+  return iscnMsgLib
+}
+
+async function getLikeNFTMsgLib () {
+  if (!likenftMsgLib) {
+    likenftMsgLib = await import(/* webpackChunkName: "likenft-msg" */ '@likecoin/iscn-js/dist/messages/likenft')
+  }
+  return likenftMsgLib
+}
+
+async function getCosmLib () {
+  if (!cosmLib) {
+    cosmLib = await import(/* webpackChunkName: "cosmjs" */ '@cosmjs/stargate')
+  }
+  return cosmLib
+}
+
+export async function getSigningClient (): Promise<any> {
   if (!iscnSigningClient) {
     const { RPC_URL } = useRuntimeConfig().public
+    const { ISCNSigningClient } = await getISCNLib()
     const c = new ISCNSigningClient()
     await c.connect(RPC_URL)
     iscnSigningClient = c
   }
   return iscnSigningClient
 }
 
-export async function getSigningClientWithSigner (signer: OfflineSigner): Promise<ISCNSigningClient> {
+export async function getSigningClientWithSigner (signer: OfflineSigner): Promise<any> {
```

Comment:
Can we keep `ISCNSigningClient`?

Reply:
[🎨 Import type ISCNSigningClient](https://github.com/likecoin/nft-book-press/pull/320/commits/c2e140d19b20aa9f4a80d5ca9c1f0bc0cd51e439)